### DISCUSSION TITLE
DAI 2024-2025 - Practical work 1 - Louis Haye / Romain Holzer / Zaïd Schouwey

### DISCUSSION THREAD

**Discussion by zaidschouwey98** [heig-vd-dai-course/heig-vd-dai-course#333] [Show and tell]
# Find & Replace Tool
## Fonctionnalité principale :
- Trouver le nombre d'occurences d'un mot recherché
- Remplacer chacune d'elle par un autre mot 
- Split un texte en plusieurs fichiers en fonction du nombre de lignes
## Fonctionnalité secondaire
- Afficher le nombre de mots/character total
- Transformer le texte en lowercase et inversément

## Repository link
https://github.com/zaidschouwey98/Find-Replace-Tool

**Comment by GeraudSilvestri** [comment]
C'est tout bon pour moi, bon codage :)

**Comment by Artotrogo** [comment]
https://github.com/zaidschouwey98/Find-Replace-Tool
cfdecf9d009b3add90bde0878356dd4d9c7d7bec
